src/app.rs

```diff
@@ -572,6 +580,15 @@ lazy_static! {
     };
 }
 
+fn validate_line_number_width(s: String) -> Result<(), String> {
+    if s.starts_with("0") {
+        Err(String::from("Custom padding characters are currently not supported. \
+        Please enter only a numeric value."))
```

Comment:
Sorry to drag this out, but can you add a test for this error? There should be other examples of tests in `tests/tests.rs` that check that ripgrep errors out.